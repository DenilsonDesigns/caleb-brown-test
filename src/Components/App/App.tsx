import MainView from "../MainView/MainView";
import NavBar from "../NavBar/NavBar";
import "./App.css";
import { Container } from "reactstrap";

const App = (): JSX.Element => {
  return (
    <div>
      <NavBar />
      <Container>
        <MainView />
      </Container>
    </div>
  );
};

export default App;
