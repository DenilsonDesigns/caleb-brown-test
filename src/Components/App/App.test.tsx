import { render } from "@testing-library/react";
import App from "./App";
import { BrowserRouter } from "react-router-dom";

describe("App:", () => {
  let app = render(
    <BrowserRouter>
      <App />
    </BrowserRouter>
  );
  it("matches snapshot", () => {
    expect(app).toMatchSnapshot();
  });
});
