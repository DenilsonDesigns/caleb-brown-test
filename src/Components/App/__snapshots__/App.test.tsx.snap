// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App: matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div>
          <nav
            class="navbar navbar-expand-md"
            style="background-color: rgb(52, 58, 64);"
          >
            <a
              class="text-light navbar-main"
              href="/main-view"
            >
              Caleb & Brown
            </a>
            <a
              class="text-light navbar-secondary"
              href="/main-view"
            >
              Dashboard
            </a>
            <a
              class="text-light navbar-secondary"
              href="/trending"
            >
              Trending
            </a>
          </nav>
        </div>
        <div
          class="container"
        >
          <div>
            <div
              class="row"
              style="margin-bottom: 1%; margin-top: 1%;"
            >
              <div
                class="col-md-3"
              >
                <h4>
                  Cryptocurrency Dashboard
                </h4>
              </div>
              <div
                class="col-md-3"
              >
                <div
                  class="input-group"
                >
                  <div
                    class="input-group-prepend"
                  >
                    <span
                      class="input-group-text"
                    >
                      <svg
                        fill="none"
                        height="1.42em"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="2"
                        viewBox="0 0 24 24"
                        width="1.42em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <circle
                          cx="11"
                          cy="11"
                          r="8"
                        />
                        <line
                          x1="21"
                          x2="16.65"
                          y1="21"
                          y2="16.65"
                        />
                      </svg>
                    </span>
                  </div>
                  <input
                    class="form-control"
                    placeholder="Type to search..."
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
            <span
              class="css-q3o1l2"
            />
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div>
        <nav
          class="navbar navbar-expand-md"
          style="background-color: rgb(52, 58, 64);"
        >
          <a
            class="text-light navbar-main"
            href="/main-view"
          >
            Caleb & Brown
          </a>
          <a
            class="text-light navbar-secondary"
            href="/main-view"
          >
            Dashboard
          </a>
          <a
            class="text-light navbar-secondary"
            href="/trending"
          >
            Trending
          </a>
        </nav>
      </div>
      <div
        class="container"
      >
        <div>
          <div
            class="row"
            style="margin-bottom: 1%; margin-top: 1%;"
          >
            <div
              class="col-md-3"
            >
              <h4>
                Cryptocurrency Dashboard
              </h4>
            </div>
            <div
              class="col-md-3"
            >
              <div
                class="input-group"
              >
                <div
                  class="input-group-prepend"
                >
                  <span
                    class="input-group-text"
                  >
                    <svg
                      fill="none"
                      height="1.42em"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                      viewBox="0 0 24 24"
                      width="1.42em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <circle
                        cx="11"
                        cy="11"
                        r="8"
                      />
                      <line
                        x1="21"
                        x2="16.65"
                        y1="21"
                        y2="16.65"
                      />
                    </svg>
                  </span>
                </div>
                <input
                  class="form-control"
                  placeholder="Type to search..."
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <span
            class="css-q3o1l2"
          />
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
